### PR TITLE
RF07 Notebook + .yaml file

### PR DESCRIPTION
**Pull request by MaxRing96**

### CODE REVIEW

**Comment by MaxRing96** [comment]
Updated RF07 Notebook and RF07_info.yaml file (two separate legs around wiggle-manoeuvre)

File: flight_phase_files/RF07_info.yaml [hunk 1]
```diff
@@ -47,17 +47,21 @@
     start: 2020-01-31 18:52:58
     end: 2020-01-31 19:06:16
   - kind: radar_cal_tilted
-    name: radar_cal_tilted
+    name: radar_cal_tilted (10deg) around Ron Brown
```
Comment (marc-prange) [inline flight_phase_files/RF07_info.yaml]: Remove additional comment after "radar_cal_tilted". "tilted" is sufficiently specific here. Additional information is only required for the straight legs for now, e.g. "straight_leg_towards_ron_brown".

**Review by marc-prange — CHANGES_REQUESTED** [review]
Flight segments look good, also the straight leg / radar calibration issue is solved. 
But some flight segments name should be adjusted, see comment in yaml file.

**Comment by MaxRing96** [comment]
Update RF07 Notebook and RF07_info.yaml (corrected segment names), and RF03 (new .to_yaml methode without dirname).

File: flight_phase_files/RF07_info.yaml [hunk 2]
```diff
@@ -1,88 +1,88 @@
...
+- kind: circle_break
+  name: cb2
+  start: 2020-01-31 17:36:28
+  end: 2020-01-31 17:52:48
+- kind: circle
+  name: c3
+  start: 2020-01-31 17:52:48
+  end: 2020-01-31 18:47:49
+- kind: straight_leg
+  name: leg1
```
Comment (marc-prange) [inline flight_phase_files/RF07_info.yaml]: Specify leg more.

File: flight_phase_files/RF07_info.yaml [hunk 3]
```diff
@@ -1,88 +1,88 @@
...
+- '2nd set of circles: Malnourished spine of a fish.'
+- 'Excursion: Partly successful with the 10deg bank (or 540deg heading) manoeuvre.'
+- The Flo-manoeuvre (wiggle) failed as the radar stopped right before it.
+segments:
+- kind: circle
+  name: c1
+  start: 2020-01-31 15:32:18
+  end: 2020-01-31 16:24:43
+- kind: circle_break
+  name: cb1
```
Comment (marc-prange) [inline flight_phase_files/RF07_info.yaml]: I suggest new convention for names of circle breaks. In this case: cb1to2

**Review by marc-prange — CHANGES_REQUESTED** [review]
Radar calibration formats are fine now. Still add specification on legs in name parameters. See flights of RF04 / RF05 for examples. I suggest a new format for circle break names, because by just indexing them (cb1, cb2, ...) it is unclear to which circles they relate (e.g. cb3 typically relates to c4 and c5, while c1 relates to c1 and c2).

**Comment by MaxRing96** [comment]
Updated the leg- and circlebreak-names.

**Review by marc-prange — APPROVED** [review]
Looks good!

**Pull request merged**